rest/src/main/java/org/jboss/pnc/rest/trigger/BuildTriggerer.java

```diff
@@ -144,4 +170,56 @@ private BuildConfigurationAudited getLatestAuditedBuildConfiguration(Integer bui
         }
         return buildConfigRevs.get(0);
     }
+    
+    /*
+     * TODO: Do not ignore certificates, rather setup servers properly.
+     */
+    private void signalBpmEvent(String uri) {
```

Comment:
Found out that we already have https://github.com/project-ncl/pnc/blob/master/common/src/main/java/org/jboss/pnc/common/util/HttpUtils.java. I suppose it would be better to reuse this class rather than creating a new one.